hudi-flink/src/main/java/org/apache/hudi/streamer/HoodieFlinkStreamer.java

```diff
@@ -18,10 +18,16 @@
 
 package org.apache.hudi.streamer;
 
+import org.apache.flink.streaming.api.datastream.DataStream;
```

Comment:
It would be better to split this statement with hudi.

Reply:
ok. I have changed it,please review.